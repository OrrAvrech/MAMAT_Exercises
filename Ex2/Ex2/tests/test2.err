Command not found.
Command not found.
Add failed: not enough parameters.
Insert failed: not enough parameters.
Command not found.
Add execution failed.
Insert execution failed.
Delay execution failed.
Depart execution failed.
Depart execution failed.
Depart execution failed.
Depart execution failed.
Depart execution failed.
Depart execution failed.
Depart execution failed.
Depart execution failed.
Depart execution failed.
Depart execution failed.
Depart execution failed.
Remove execution failed.
Remove execution failed.
Remove execution failed.
Remove execution failed.
Remove execution failed.
