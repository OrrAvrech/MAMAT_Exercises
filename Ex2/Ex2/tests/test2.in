


Print
print
add
Add
Insert
insert
Add 1 D AAA R
Insert 1 D
Insert 1 D
Insert 2 D
Insert 3 I
Insert 4 I
Print
Add 1 D AAA R
Add 2 D AAA R
Add 3 I AAA R

Add 4 I AAA R
Print
Add 5 D BBB E
Add 6 I BBB E
Add 7 D CCC R
Add 8 I CCC R
Print
Delay AAA


Print
Depart
Depart
Depart
Print
Remove 4
Print
Change AAA KFC
Print
Depart
Depart
Print
Delay AAA
Print
Delay KFC
Delay KfC
Print
Depart
Depart
Depart
Depart
Depart
Print
Depart
Depart
Depart
Print
Print
Depart
Depart
Depart
Depart
Print
Remove 1
Remove 1
Remove 2
Remove 2
Print
Remove 10
Remove -1
Remove 3
Remove 4
Print

Exit
Print