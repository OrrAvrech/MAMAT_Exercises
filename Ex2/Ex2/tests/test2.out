Airport status:

Airport status:
Runway 1 domestic
0 flights are waiting:
Runway 2 domestic
0 flights are waiting:
Runway 3 international
0 flights are waiting:
Runway 4 international
0 flights are waiting:

Airport status:
Runway 1 domestic
1 flights are waiting:
Flight 1 D AAA R
Runway 2 domestic
1 flights are waiting:
Flight 2 D AAA R
Runway 3 international
1 flights are waiting:
Flight 3 I AAA R
Runway 4 international
1 flights are waiting:
Flight 4 I AAA R

Airport status:
Runway 1 domestic
2 flights are waiting:
Flight 5 D BBB E
Flight 1 D AAA R
Runway 2 domestic
2 flights are waiting:
Flight 2 D AAA R
Flight 7 D CCC R
Runway 3 international
2 flights are waiting:
Flight 6 I BBB E
Flight 3 I AAA R
Runway 4 international
2 flights are waiting:
Flight 4 I AAA R
Flight 8 I CCC R

Airport status:
Runway 1 domestic
2 flights are waiting:
Flight 5 D BBB E
Flight 1 D AAA R
Runway 2 domestic
2 flights are waiting:
Flight 7 D CCC R
Flight 2 D AAA R
Runway 3 international
2 flights are waiting:
Flight 6 I BBB E
Flight 3 I AAA R
Runway 4 international
2 flights are waiting:
Flight 8 I CCC R
Flight 4 I AAA R

Airport status:
Runway 1 domestic
0 flights are waiting:
Runway 2 domestic
2 flights are waiting:
Flight 7 D CCC R
Flight 2 D AAA R
Runway 3 international
1 flights are waiting:
Flight 3 I AAA R
Runway 4 international
2 flights are waiting:
Flight 8 I CCC R
Flight 4 I AAA R

Airport status:
Runway 1 domestic
0 flights are waiting:
Runway 2 domestic
2 flights are waiting:
Flight 7 D CCC R
Flight 2 D AAA R
Runway 3 international
1 flights are waiting:
Flight 3 I AAA R

Airport status:
Runway 1 domestic
0 flights are waiting:
Runway 2 domestic
2 flights are waiting:
Flight 7 D CCC R
Flight 2 D KFC R
Runway 3 international
1 flights are waiting:
Flight 3 I KFC R

Airport status:
Runway 1 domestic
0 flights are waiting:
Runway 2 domestic
0 flights are waiting:
Runway 3 international
1 flights are waiting:
Flight 3 I KFC R

Airport status:
Runway 1 domestic
0 flights are waiting:
Runway 2 domestic
0 flights are waiting:
Runway 3 international
1 flights are waiting:
Flight 3 I KFC R

Airport status:
Runway 1 domestic
0 flights are waiting:
Runway 2 domestic
0 flights are waiting:
Runway 3 international
1 flights are waiting:
Flight 3 I KFC R

Airport status:
Runway 1 domestic
0 flights are waiting:
Runway 2 domestic
0 flights are waiting:
Runway 3 international
0 flights are waiting:

Airport status:
Runway 1 domestic
0 flights are waiting:
Runway 2 domestic
0 flights are waiting:
Runway 3 international
0 flights are waiting:

Airport status:
Runway 1 domestic
0 flights are waiting:
Runway 2 domestic
0 flights are waiting:
Runway 3 international
0 flights are waiting:

Airport status:
Runway 1 domestic
0 flights are waiting:
Runway 2 domestic
0 flights are waiting:
Runway 3 international
0 flights are waiting:

Airport status:
Runway 3 international
0 flights are waiting:

Airport status:

